app/views/dish/show.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<h1> <%= @dish.name %>  </h1>
+
+<p> <%= @dish.description %> </p>
+
+<h3> Dish Ingredients </h3>
+
+<div id="dish_ingredient_list">
+  <% @dish.ingredients.each do |ingredient| %>
+    <p> <%= ingredient.name %> </p>
+  <% end %>
+</div>
+
+<h3> Calorie Count </h3>
+<p> Calories: <%= @dish_calories %> </p>
+
+<h2>Add Existing Ingredient To Dish</h2>
+<%= form_with model: @dish, local: true do |form| %>
+  <div>
+    <%= form.label :ingredient_id, "Ingredient ID" %>
+    <%= form.select :ingredient_id, Ingredient.all.map { |ingredient| [ingredient.name, ingredient.id] }, { include_blank: true } %>
+    <%= form.submit "Submit", form: { action: dish_path(@dish) } %>
+  </div>
+<% end %>
```

Comment:
This form has some interesting things going on. I do like the fact that you used a "select" over a text field, but I would have like to see the logic you have there to populate the dropdown moved to a helper method on the model. For the submit button, what made it necessary for you to add this: `form: { action: dish_path(@dish) }`?